<launch>
	<node pkg="dji_sdk" type="dji_sdk_node" name="dji_sdk" output="screen">
		<!-- node parameters -->
                <param name="drone_version" type="string" value="M100"/>
		<param name="serial_name" type="string" value="/dev/ttyUSB0"/>
		<param name="baud_rate" type="int" value="230400"/>
		<param name="app_id" type="int" value="1026762"/>
		<param name="app_version" type="int" value="1"/>
		<param name="app_bundle_id" type="string" value="Welcome to use dji-sdk"/>
		<param name="enc_key" type="string" value="1b838efe0d1b733e9015c750b2250fd61eb75dcbc3b309bf16813bd57294ed36"/>
		<param name="groundstation_enable" type="int" value="1"/>
	</node>
</launch>
